Exiled.API/Features/Intercom.cs

```diff
@@ -0,0 +1,59 @@
+// -----------------------------------------------------------------------
+// <copyright file="Intercom.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    /// <summary>
+    /// A set of tools to easily handle the Intercom.
+    /// </summary>
+    public static class Intercom
+    {
+        /// <summary>
+        /// Gets or sets the text displayed on the intercom screen.
+        /// </summary>
+        public static string DisplayText
+        {
+            get => global::Intercom.host.CustomContent;
+            set => global::Intercom.host.CustomContent = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the current state of the intercom.
+        /// </summary>
+        public static global::Intercom.State State
+        {
+            get => global::Intercom.host.IntercomState;
+            set => global::Intercom.host.IntercomState = value;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether or not the intercom is currently being used.
+        /// </summary>
+        public static bool InUse => State == global::Intercom.State.Transmitting || State == global::Intercom.State.TransmittingBypass || State == global::Intercom.State.AdminSpeaking;
+
+        /// <summary>
+        /// Gets the <see cref="Player"/> that is using the intercom. Will be <see langword="null"/> if <see cref="InUse"/> is <see langword="false"/>.
+        /// </summary>
+        public static Player Speaker => !InUse ? null : Player.Get(global::Intercom.host.speaker);
+
+        /// <summary>
+        /// Gets or sets the remaining speech time of the intercom.
+        /// </summary>
+        public static float SpeechRemainingTime
+        {
+            get => !InUse ? 0f : global::Intercom.host.speechRemainingTime;
```

Comment:
Return the real value

Reply:
@joker-119 any reasons for that

Reply:
Because if it's not in use the remaining time should be 0, since it's not being used. Means we don't need to do checks like
if (Intercom.InUse && Intercom.SpeechRemainingTime > 10f)

We can simply if (Intercom.SpeechRemainingTime > 10f)